src/main/java/org/lobid/resources/ElasticsearchIndexer.java

```diff
@@ -313,12 +316,13 @@ private String enrich(final String index, final String queryField,
 								.must(new MultiMatchQueryBuilder(query.first)
 										.fields(WikidataGeodata2Es.FIELD_BOOST)
 										.type(MultiMatchQueryBuilder.Type.CROSS_FIELDS)
-										.operator(Operator.AND))
+										.operator(Operator.AND));
+					if (query.second != "")
```

Comment:
I think testing with `!isEmpty()` should be safer here due to the reference comparison done by `!=`.